client/pkg/omnictl/config/config.go

```diff
@@ -9,16 +9,22 @@ import (
 	"errors"
 	"fmt"
 	"os"
+	"path/filepath"
 
 	"github.com/adrg/xdg"
+	"github.com/siderolabs/go-api-signature/pkg/fileutils"
+	"github.com/siderolabs/talos/pkg/machinery/client/config"
+	"github.com/siderolabs/talos/pkg/machinery/constants"
 	"gopkg.in/yaml.v3"
 )
 
 const (
 	// OmniConfigEnvVar is the environment variable to override the default config path.
 	OmniConfigEnvVar = "OMNICONFIG"
 
-	relativePath       = "omni/config"
+	omniConfigFolder   = "omni"
```

Comment:
I recommend using "directory" as folder is the Windows term :) omniConfigDir should be fine here